src/Symfony/Bridge/Doctrine/DataCollector/DoctrineDataCollector.php

```diff
@@ -121,6 +123,38 @@ public function getName()
         return 'db';
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function getCasters()
+    {
+        return parent::getCasters() + [
+                ObjectParameter::class => static function (ObjectParameter $o, array $a, Stub $s): array {
```

Comment:
Too much indentation here. It should be 4 spaces.

Reply:
Looks like fabbot is not detecting this.